estuary/models/productversion.py

```diff
@@ -0,0 +1,9 @@
+from neomodel import StringProperty
+
+from estuary.models.base import EstuaryStructuredNode
+
+
+class ProductVersion(EstuaryStructuredNode):
+    """Definition of RHEL Product Version in Neo4j."""
```

Comment:
Sorry, I missed this before. I think it'd be best if this was just a generic concept of a product version.
```suggestion
    """Definition of Product Version in Neo4j."""
```